presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt

```diff
@@ -1,126 +1,286 @@
 #$ TITLE: SupaflyFPV Freestyle 5 Inch
 #$ FIRMWARE_VERSION: 4.3
 #$ CATEGORY: TUNE
-#$ STATUS: OFFICIAL
+#$ STATUS: EXPERIMENTAL
 #$ KEYWORDS: Supafly, SupaflyFPV, 4S, 6S, Freestyle, 5 inch, 5"
 #$ AUTHOR: SupaflyFPV
-#$ DESCRIPTION: SupaflyFPV Community Freestyle Preset for 5 Inch Builds
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION: <a href="https://supaflyfpv.com"><img src="https://supaflyfpv.com/wp-content/uploads/2022/06/Logo-Cropped192.svg" width="220px" style="margin-left: auto; margin-right: auto; display: block;"/></a>
 #$ DESCRIPTION:
-#$ DESCRIPTION: Select the option for your build with or without HD camera (eg GoPro) as the baseline.
+#$ DESCRIPTION: <br>
```

Comment:
@limon does a empty #$ DESCRIPTION: line add a line feed?

We should have a better way to add <br> break tags. Can we add this to remove duplicate lines please?

Reply:
I haven't figured this part. If it's just regular text - it does for sure. When when it's marked parcer, it works kinda weirdly and sometimes it does, sometimes it doesn't...

Reply:
If you want me to do it another way let me know but this one delivers the desired formatting...

Reply:
@SupaflyFPV  as long as you are satisfied with how it looks in the configurator :)